xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1313,7 +1317,7 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
     else
         ios_log(this, logfp, "\n=== Interval %d stats ===", interval);
     ios_log(this, logfp, "      Duration : %" PRIu64 " secs",
-            (uint64_t)(now - stats->started_at));
+            (uint64_t)(conf->last_fop_hit - stats->started_at));
```

Comment:
This could be misleading since the interval duration can be smaller than the real time elapsed between interval stats requests.

Maybe we could add the start and end times considered for the interval.

Reply:
done